webserver/src/macros_local.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_export]
+macro_rules! get_db_connection {
+    ($pool:expr) => {{
```

Comment:
Are macros necessary. Might make the code hard to read for newer people.

Reply:
Not necessary, just cleaner and shorter.